app/api/module/Api/src/Entity/Application/Application.php

```diff
@@ -2365,6 +2526,72 @@ public function canAddFinancialEvidence()
         return true;
     }
 
+    public function canAddPsvSmallEvidence()
+    {
+        // This only applies to PSV licences that are not special restricted
+        if (!$this->isPsv() || $this->isSpecialRestricted()) {
```

Comment:
comment indicates could use check on special restricted on LHS  performance would be better right but not a blocker due to short circuit